app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,74 @@
+class ReviewsController < ApplicationController
+  before_action :find_restaurant , only: [:create]
+
+  def index
+    render :json => Review.all.to_json(:except => [:created_at, :updated_at, :id, :restaurant_id])
+  end
+
+  def new
+    @review = Review.new
+  end
+
+  def create
+    puts "creating a review #{params["review"]}"
+    # Parameters: {"review"=>{"name"=>"dsdas", "comment"=>"dasdas", "rating"=>4, "restaurant_id"=>2}}
+    @review = @restaurant.reviews.create(review_params)
```

Comment:
I'm pretty sure create performs both new and save, so calling save again in the next line is redundant.
Change the create to new so that the save is the first time save is performed.